netlify.toml

```diff
@@ -2,9 +2,6 @@
 publish = "public"
 command = "make production-build"
 
-[build.environment]
-HUGO_VERSION = "0.75.1"
```

Comment:
Can I get a little context on this PR and why it's necessary? 

Not opposed to removing this, but it does seem to be causing a build failure.

Reply:
https://github.com/cncf/hugo-netlify-starter/issues/68 might be helpful for context, but maybe @chalin could offer more insight?

Reply:
Yes, #68 provides the context. Thought we can discuss on Wednesday if you'd like more.

Reply:
Nope, tysm <3